sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/BlobsCheckpointStore/BlobsCheckpointStoreTests.cs

```diff
@@ -36,5 +51,301 @@ public void ConstructorRequiresRetryPolicy()
         {
             Assert.That(() => new BlobsCheckpointStore(Mock.Of<BlobContainerClient>(), null), Throws.ArgumentNullException);
         }
+
+        [Test]
+        public async Task ListOwnershipLogsStartAndComplete()
+        {
+            var blobList = new List<BlobItem>{
+                BlobsModelFactory.BlobItem($"{FullyQualifiedNamespace}/{EventHubName}/{ConsumerGroup}/ownership/{Guid.NewGuid().ToString()}",
```

Comment:
You're definitely more thorough than we've been with respect to validating logging.  I'm a bit on the fence; while I think that its great, I'm concerned about potentially setting a bar for rigor around logging that we may not want to apply consistently.

For now, I think I'm content to leave things as you've written.   I'll think more on whether we should be doing the same across other areas.

Reply:
I've been bitten by too many event source logging bugs in my time, perhaps :)